css/main.css

```diff
@@ -232,81 +225,98 @@ footer {
     bottom: 0;
 }
 
-footer a p {
+footer i p {
     display: inline;
 }
 
-
-#myBtn {
-    display: none;
-    position: fixed;
-    bottom: 20px;
-    right: 30px;
-    z-index: 99;
-    font-size: 36px;
-    border: none;
-    outline: none;
-    background-color: rgba(95, 95, 95, 0.74);
-    color: white;
-    cursor: pointer;
-    padding: 12px 25px;
-    border-radius: 20%;
-  }
-  
-  #myBtn:hover {
-    background-color: #555;
-  }
-
-@media (max-width: 730px) {
-    .menu li {
-        width: 100%;
-        justify-content: center;
-        max-height: 80px;
-        overflow: hidden;
-    }
+footer i {
+    padding-right: 10px;
 }
 
-@media (max-width: 729px) {
+
+@media (max-width: 760px) {
     .menu {
-        height: 20px;
         width: 100%;
         display: flex;
         justify-content: flex-end;
-        align-content: center;
-        padding: 10px;
+        align-content: stretch;
         box-sizing: border-box;
-        transform: translateY(100%);
+        top: 0;
     }
 
-    .nav-trigger {
-        border: 0;
+    #nav-trigger {
```

Comment:
czy te style muszą być w media-quieries?
nie możemy ich dodać poza media-queries a w media quieries dać tylko to co chcemy zmienić by było łatwiej zobaczyć co jest kluczowe?

Reply:
Dodałam to do media-quieries ponieważ jest to element widoczny tylko dla małych rozdzielczości. Po przesunięciu poza, wyświetla się również przy dużej rozdzielczości.